Dockerfile

```diff
@@ -1,7 +0,0 @@
-FROM nfcore/base
```

Comment:
Did you accidentally delete your Dockerfile maybe? If you are not using DSL2 you still need it.

Reply:
Ok I see you are trying to use the containers from the base.config

Reply:
Now there is a single docker container.

Reply:
ok fine with me, if @Phil agrees to have it this way